bosh/releases/pre_render_scripts/api/cc_uploader/patch_pre-start.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+target="/var/vcap/all-releases/jobs-src/capi/cc_uploader/templates/pre-start.erb"
+
+# Remove sysctl calls as we are running in containers.
```

Comment:
It would be better to point to the other script too, as better justification.

Reply:
Addressed on https://github.com/SUSE/scf/pull/2628.